api/app.js

```diff
@@ -16,9 +17,12 @@ app.use(express.json());
 app.use(express.static(path.join(__dirname, "public")));
 
 
+
+// JP. I'm a bit unser about how we're going to fire both controllers. I think we need to think about mixing them together 
```

Comment:
I think we can get away with having two controllers which get invoked via the same submit button, so this configuration currently you have is pretty good